lib/ansible/modules/cloud/amazon/s3_bucket.py

```diff
@@ -374,6 +439,19 @@ def wait_payer_is_applied(module, s3_client, bucket_name, expected_payer, should
         return None
 
 
+def wait_encryption_is_applied(module, s3_client, bucket_name, expected_encryption):
+    for dummy in range(0, 12):
+        try:
+            encryption = get_bucket_encryption(s3_client, bucket_name)
+        except (BotoCoreError, ClientError) as e:
+            module.fail_json_aws(e, msg="Failed to get updated encryption for bucket")
+        if encryption != expected_encryption:
+            time.sleep(5)
+        else:
+            return encryption
+    module.fail_json(msg="Bucket encryption failed to apply in the excepted time")
```

Comment:
I haven't run the looped tests on this PR to see how frequent the failures are, but each time I've rerun shippable it has been unstable. I think the allotted wait time might need to be increased for encryption. Also s/excepted/accepted? In other places too.

As this pattern is done in a number of places, would it make sense to turn the pattern into a wrapper that would take the method, method params, and the expected value?

Reply:
I suspect s/excepted/expected/